README.md

```diff
@@ -2,12 +2,58 @@
 
 [![Join the chat at https://codercampslackin.azurewebsites.net/](https://codercampslackin.azurewebsites.net/badge.svg)](https://codercampslackin.azurewebsites.net/)
 
-## Installing Jekyll
+To get started developing, you have two options. You can develop in a Docker Container using VS Code, or you can install Ruby locally and develop locally.
+
+## Installing Ruby and Jekyll
+
+Do the following if you want to develop using Ruby on your local machine.
 
 1. Install Ruby 2.1 ([Linux](https://www.ruby-lang.org/en/documentation/installation/), [Mac](https://gorails.com/setup/osx/10.10-yosemite), [Windows](http://rubyinstaller.org/))
 2. If you are on Windows, install [DevKit](http://rubyinstaller.org/add-ons/devkit/)
 3. `gem install bundler`
 
```

Comment:
should I add a section for running development locally without VS code?

Reply:
That would be great. It should just be a few extra commands right?